scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -1052,7 +1052,7 @@ class ScalaTestAntTask extends Task {
   // Class to hold data from tagsToInclude and tagsToExclude elements.
   //
   private class TextElement {
-      private var text: String = null
+      private var text: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.